File: Java/Graph.java
import java.util.Scanner;

class Graph {
    private int vertices;
    private int[][] adjacencyMatrix;

    public Graph(int vertices) {
        this.vertices = vertices;
        adjacencyMatrix = new int[vertices][vertices];
        for (int i = 0; i < vertices; i++) {
            for (int j = 0; j < vertices; j++) {
                adjacencyMatrix[i][j] = 0;
            }
        }
    }

    public void addEdge(int src, int dest) {
        adjacencyMatrix[src][dest] = 1;
        adjacencyMatrix[dest][src] = 1;
    }

    public void printGraph() {
        System.out.println("Adjacency Matrix:");
        for (int i = 0; i < vertices; i++) {
            for (int j = 0; j < vertices; j++) {
                System.out.print(adjacencyMatrix[i][j] + " ");
            }
            System.out.println();
        }
    }
}

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the number of vertices: ");
        int vertices = scanner.nextInt();

        Graph graph = new Graph(vertices);
        int option;

        do {
            System.out.println("Graph Options");
            System.out.println("1. Add edge");
            System.out.println("2. Print graph");
            System.out.println("3. Exit");
            System.out.print("Enter your choice: ");
            option = scanner.nextInt();

            switch (option) {
                case 1:
                    System.out.print("Enter source vertex: ");
                    int src = scanner.nextInt();
                    System.out.print("Enter destination vertex: ");
                    int dest = scanner.nextInt();
                    graph.addEdge(src, dest);
                    break;
                case 2:
                    graph.printGraph();
                    break;
                case 3:
                    System.out.println("Exiting...");
                    break;
                default:
                    System.out.println("Invalid option");
            }

        } while (option != 3);
    }
}

File: Kotlin/Graph.kt
import java.util.*

class Graph(private val vertices: Int) {
    private val adjacencyMatrix: Array<IntArray> = Array(vertices) { IntArray(vertices) }

    init {
        for (i in 0 until vertices) {
            for (j in 0 until vertices) {
                adjacencyMatrix[i][j] = 0
            }
        }
    }

    fun addEdge(src: Int, dest: Int) {
        adjacencyMatrix[src][dest] = 1
        adjacencyMatrix[dest][src] = 1
    }

    fun printGraph() {
        println("Adjacency Matrix:")
        for (i in 0 until vertices) {
            for (j in 0 until vertices) {
                print("${adjacencyMatrix[i][j]} ")
            }
            println()
        }
    }
}

fun main() {
    val scanner = Scanner(System.`in`)
    print("Enter the number of vertices: ")
    val vertices = scanner.nextInt()

    val graph = Graph(vertices)
    var option: Int

    do {
        println("Graph Options")
        println("1. Add edge")
        println("2. Print graph")
        println("3. Exit")
        print("Enter your choice: ")
        option = scanner.nextInt()

        when (option) {
            1 -> {
                print("Enter source vertex: ")
                val src = scanner.nextInt()
                print("Enter destination vertex: ")
                val dest = scanner.nextInt()
                graph.addEdge(src, dest)
            }
            2 -> {
                graph.printGraph()
            }
            3 -> println("Exiting...")
            else -> println("Invalid option")
        }

    } while (option != 3)
}
